### DISCUSSION TITLE
欢迎来到 nestjs 中文社区

### DISCUSSION THREAD

**Discussion by zuohuadong** [nestcn/docs.nestjs.cn#249] [1-公告]
## 👋 欢迎来到 nestjs 中文社区
  * 提问问题.
  * 分享 idea.
  * 友善交流.

**Comment by dyb881** [comment]
我有个不错的想法，欢迎大家给点意见
https://github.com/dyb881/nest-serve

**Comment by Mingyang-Li** [comment]
@zuohuadong  各位大佬好，我这里想了解一下 Nest.js 的文档还有哪些部分需要翻译，我愿意有空的时候帮大家翻译一部分

**Reply by zuohuadong** [reply]
> @zuohuadong 各位大佬好，我这里想了解一下 Nest.js 的文档还有哪些部分需要翻译，我愿意有空的时候帮大家翻译一部分

https://github.com/nestcn/docs.nestjs.cn/discussions/252

8 还有很多部分
